Feature/card 0039/improve-cutoff-grade-page-website

JSantosMoraes opened the pull request:
# [📝 Card-0039 / Aprimorar Página Notas de Corte ENEM - Site ORIENTA](https://app.clickup.com/t/86abc6uxx)

## 🎯 Objetivo da Task
O objetivo principal é garantir que a página de nota de corte do ENEM esteja totalmente funcional, visualmente alinhada com o restante do site e forneça todas as informações necessárias aos usuários.

## 📂 Arquivos Criados
- Nenhum arquivo foi criado.

## 🔄️ Arquivos Modificados
1. _enem-score.html_
   - Alteração na ordem dos textos seção de pesquisa.
   - Textos melhorados.
2.  _style.css_
   - O visual dos botões da seção de pesquisa foram melhorados.
   - O efeito de hover foi adicionado aos botões.
   - Alteração na cor dos botões e títulos da seção de pesquisa.
4. _main.js_
   - **Correção dos erros causados pela task 043.**
   - Mais informações de notas de corte foram adicionadas.

JSantosMoraes committed:
🐛fix: JS fix

JSantosMoraes committed:
💄style: Improvements in the search section

- Hover foi adicionado aos botões

- Melhor responsividade

- Alteração nas cores

- Valor das fontes, padding e margins alterados

JSantosMoraes committed:
🐛fix: Our Team Carousel fixed

Bug do JS que adiciona a funcionalidade do carrossel do nosso time foi corrigido.

JSantosMoraes committed:
🚧feat: Improved Information, Styling and Text

- Mais informações de notas de corte foram adicionadas.

- O estilo da página foi melhorado

- Melhoria nos textos

JSantosMoraes committed:
Merge branch 'main' into feature/card-0039/improve-cutoff-grade-page-website

Signed-off-by: Jonas Santos <149276335+JSantosMoraes@users.noreply.github.com>

JSantosMoraes committed:
📝fix: Correction in the text

Correção nas informações passadas pelo texto.

JSantosMoraes committed:
Merge branch 'feature/card-0039/improve-cutoff-grade-page-website' of https://github.com/felipebarbosa24/Projeto-ORIENTA-ETE-JBL into feature/card-0039/improve-cutoff-grade-page-website

felipebarbosa24 commented:
Olá @JSantosMoraes, estou iniciando a revisão do seu PR. Mas de antemão gostaria de destacar aquele ponto que já mencionei que é sobre fazer o Merge pelo GitHub - pelo menos pelo texto é oque acredito que fizesse -, recomendo sempre fazer o Resolve in Merge local ao invés de pelo GitHub, porque embora esteja em sua branch abre uma margem muito fácil para retrabalho, fazendo de forma local você pode facilmente abortar o merge ou simplesmente apagar o repositório e clonar denovo com as informações que puxar do remoto. Então acredito que venha a ser até uma boa prática para evitar problemas como tivemos no [Card-0043](https://github.com/felipebarbosa24/Projeto-ORIENTA-ETE-JBL/pull/28) #28 .

<img width="1097" height="73" alt="image" src="https://github.com/user-attachments/assets/caaa43fb-06fa-4605-9ddb-694e48d64ab3" />
<img width="1088" height="79" alt="image" src="https://github.com/user-attachments/assets/b37ad5fb-2134-4371-9721-a6762c09d52e" />

felipebarbosa24 commented on css/style.css:
Bom @JSantosMoraes, acredito que para manter o padrão das outras páginas seja bom acrescentar uma escrita de background, ex:
<img width="778" height="319" alt="Captura de tela 2025-10-06 234039" src="https://github.com/user-attachments/assets/16cbebad-1d01-4030-96ea-450239da809b" />

Da forma que foi feita inicialmente por @Killao1 ficou assim:
<img width="935" height="410" alt="Captura de tela 2025-10-06 234047" src="https://github.com/user-attachments/assets/6368ea9c-4c67-4a13-ae97-659a8bbbaf05" />

Então recomendo solicitar para o mesmo o template e preencher o quadro de fundo ou mudar de alguma forma que você prefira e ache que se adeque. Porém acho bom mudar somente essa parte do texto e no máximo a figura do quadro mesmo, porque os símbolos estatísticos, etc, achei bacana da forma que ele implementou.

felipebarbosa24 commented on js/main.js:
Caso possa, recomendo preencher mais informações de mais cursos pelo menos nas 5 primeiras faculdades a seguir - depois abro task para o preenchimento das demais universidades:
<img width="764" height="628" alt="image" src="https://github.com/user-attachments/assets/c0263658-16ef-434b-b619-f758aafe0b52" />

Então na tabela você põe mais cursos e a média que geralmente é necessário ter para passar - a famosa nota de corte.
<img width="1899" height="460" alt="image" src="https://github.com/user-attachments/assets/cf6b1aec-19ec-4d5f-baab-02afbdc258e1" />

Eu em conjunto de Sabrina e Victor separamos uma documentação no drive com diversos cursos e notas de corte, então caso queira se basear pode consultar no link a seguir: https://drive.google.com/drive/folders/1h2ddM96yXSBsswRDwBjtvXVxthYh7rQm?usp=drive_link

felipebarbosa24 reviewed (changes requested):
Bom @JSantosMoraes, por ora recomendo mais essas alterações a seguir:

MateusPereira1 commented:
Olá @JSantosMoraes, não encontrei grandes poblemas em seu código, apenas algumas observações mesmo, como a imagem da logo do **SISU** que recomendo centralizar a partir do primeiro ponto de quebra, que se localiza por volta dos 1200px de largura do site como se vê na imagem a seguir 
<img width="930" height="881" alt="image" src="https://github.com/user-attachments/assets/b69526a1-8894-4d25-89ed-047528adba75" />
Ou seja, resumindo, desse ponto de quebra, abaixo, que você centralize a imagem para um melhor designh visual.

MateusPereira1 commented:
Outra observação que eu julgo como interessante a nível de UX mobile, que seria colocar um **Height auto** pois ficaria menos confuso para o usuário, não é algo que vejo como importante, mas uma observação que espera a opinião também de @felipebarbosa24 @Killao1 @marcosp46, vou colocar algumas imagens pra exemplificar.
<img width="409" height="872" alt="com scroll" src="https://github.com/user-attachments/assets/d8004287-1108-4069-970b-85160616b202" />
<img width="424" height="897" alt="semscroll" src="https://github.com/user-attachments/assets/531a88e0-21e4-4ca1-8892-2650f1dcdf3d" />
Essa mudança seria bem de boa, apenas adicionando uma propiedade de **"h-auto"** já resolvia esse poblema

JSantosMoraes committed:
🚧feat: More information, better responsiveness and visuals

- Mais informações de notas de corte foram adicionadas. ( JS )

- Alteração na imagem do background.

- Algumas melhorias de visual para a versão mobile e pc.

JSantosMoraes commented:
Olá @felipebarbosa24 @MateusPereira1 @Killao1 @marcosp46. Acabei de postar as alterações requisitadas por vocês, aguardando a análise.

felipebarbosa24 reviewed (changes requested):
Olá @JSantosMoraes. Recomendo diminuir um pouco o texto de background, porque no iphone se fica nas bordas e no samsung s8+
<img width="653" height="614" alt="image" src="https://github.com/user-attachments/assets/b1ba31cd-b542-4298-bc02-62bc1084593e" />

Quanto ao preenchimento de informações da PopUp de cada universidade, achei que tá meio pobre de cursos, porque essas universidades são muito importantes e tem muito mais cursos, claro que de boa no momento não pôr todos, mas tem muitos relevantes.
<img width="1856" height="596" alt="image" src="https://github.com/user-attachments/assets/cefe6dbf-19e5-44a6-a956-2f46f413235e" />
(exemplo: ver boa parte dos cursos da ufpe no site oficial, etc: https://www.ufpe.br/cursos/graduacao)

Tem muitos cursos. Por universidade pode selecionar uns 20 ou 25. Pesquisa os principais e mais demandados.

Sei que dá um trabalhinho, mas essa task faz justamente esses aprimoramentos. As outras universidades abro task para outras pessoas.

@JSantosMoraes para compensar e não ficar muito pesado faz somente as 3 primeiras universidades.

JSantosMoraes committed:
📝feat: More course information

- Agora as três primeiras faculdades possuem entre 20 e 25 informações de cursos.

JSantosMoraes commented:
Olá @felipebarbosa24. Acabei de corrigir o que foi pedido, aguardando a revisão.

JSantosMoraes commented:
@felipebarbosa24 @MateusPereira1 @Killao1 @marcosp46

felipebarbosa24 reviewed (dismissed):
Olá @JSantosMoraes, revisando o PR identifiquei que na versão mobile a lista de faculdades os nomes até aparecem de forma ok:
<img width="480" height="610" alt="Captura de tela 2025-10-10 202137" src="https://github.com/user-attachments/assets/9b5ba9e2-97cc-468c-9807-a56fcf9243f5" />

Porém quando se pesquisa o nome da faculdade ela quebra:
<img width="481" height="933" alt="Captura de tela 2025-10-10 201912" src="https://github.com/user-attachments/assets/b9c65d12-fc16-4674-b077-476f750dd63d" />

Então recomendo dar uma checada e ver a viabilidade de resolução. Mas tirando isso, está indo muito bem na task, parábens!👏🏻

JSantosMoraes committed:
🐛fix: broken buttons on mobile

- Correção na quebra dos botões ao utilizar a barra de pesquisa na versão mobile.

JSantosMoraes commented:
Foi tranquilo @felipebarbosa24. Acabei de corrigir.

MateusPereira1 commented:
Olá @JSantosMoraes não encontrei mais erros depois da sua última commit, estou aprovando aqui, parabéns 🚀🚀

Killao1 reviewed (changes requested):
Olá, @JSantosMoraes , notei que na tela de celular, todo o campo de faculdades aparece; mostrando todas as faculdades até o fim da página. Ou seja, caso existam muitas faculdades no campo, eu só chegarei até o fim da página/footer se passar por todas elas. É besta mas fica desconfortável.

<img width="397" height="748" alt="image" src="https://github.com/user-attachments/assets/57f5c81a-78c6-4a4d-ac1f-9ef1ea014ff0" />

Creio que um overflow-y: auto; no @query já resolveria.

JSantosMoraes commented:
Olá, @Killao1! Essa funcionalidade de rolagem da lista foi removida no mobile a pedido do @MateusPereira1 anteriormente. Vocês preferem que permaneça como está agora ou que o scroll seja recolocado na versão mobile?

@felipebarbosa24 @MateusPereira1 @Killao1 @marcosp46

Killao1 reviewed (approved):
Entendi o que você quis dizer, @JSantosMoraes . Então não encontrei nenhum erro/inconsistência no seu código.

JSantosMoraes committed:
📝fix: Text correction

MateusPereira1 reviewed (approved):
Só aprovando novamente @JSantosMoraes, boa task, parabéns !!!
